Makefile

```diff
@@ -175,6 +175,14 @@ push-bench-images: install-bench
 	docker tag pachyderm_test pachyderm/bench:`git rev-list HEAD --max-count=1`
 	docker push pachyderm/bench:`git rev-list HEAD --max-count=1`
 
+tag-images:
+	docker tag pachyderm_pachd pachyderm/pachd:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
+	docker tag pachyderm_worker pachyderm/worker:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
+
+push-images: tag-images
+	docker push pachyderm/pachd:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
+	docker push pachyderm/worker:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
+
```

Comment:
`make docker-build` tags the images as `*:local` but I needed a way to tag and push images with the commit version, so I could test a particular build in the cloud.  `push-bench-images` kinda does that but it also tries to build and push the benchmark image, which is not relevant for what I was trying to do.